scripts/preprocessing/04b-run_ssp.py

```diff
@@ -52,14 +53,17 @@ def run_ssp(cfg, subject, session=None):
         raw_fname_in.update(split='01')
 
     raw = mne.io.read_raw_fif(raw_fname_in)
-    # XXX : n_xxx should be options in config
     msg = 'Computing SSPs for ECG'
     logger.debug(gen_log_message(message=msg, step=4, subject=subject,
                                  session=session))
 
+    reject_ecg_ = cfg._get_reject(
+        reject=cfg.ssp_reject_ecg,
+        epochs=(create_ecg_epochs(raw)
+                if cfg.ssp_reject_ecg == 'autoreject_global' else None))
```

Comment:
I'd suggest to add a new public function `get_ssp_reject()` to `config.py`, and calling that outside of the processing function when creating `cfg`:

```python
cfg = BunchConst(...,
    ssp_reject_ecg=config.get_reject_ssp(which='ecg')
)
```

Reply:
Not sure I understand this reasoning

Reply:
We recently switched to the script-local cfg objects that should contain all the settings required for that particular script (and only those)

Anything "constant" should be added to that object before running the processing functions, which will receive `cfg` as a parameter

cc @agramfort

Reply:
Reason is we've started to try and make our functions as "pure" as possible to avoid a total mess of side-effects

Reply:
but you have that bunch constant already there and it is used, what is the point?

Reply:
What you suggest is less readable than the current line defining the bunch constant; on the other hand we need the logic in the lines you commented on to pass the ecg/eog epochs computed from the raw, nothing that can be handled at config level.